src/RestEndpointHandlers.cpp

```diff
@@ -55,6 +59,69 @@ static shared_ptr<http_response> const mapErrorToResponse(Error const &err) {
   return make_shared<string_response>(responseBody.dump(), statusCode);
 }
 
+//
+// Helper macros
+//
+
+#define REQUIRED_STRING_FIELD(name, body)                                      \
```

Comment:
macros are probably not the most beautiful solution here..
however, as per our discussion, there are valid reasons to keep them as implemented here.

maybe just add a prefix `PARSE_ ` to the macros, so the macro name gives a little more information about its' function.